### ISSUE TITLE
Showing wrong 64-bit integer values

### ISSUE THREAD

**Issue by ageraldy** [lazierthanthou/sqlite-manager#15]
When I enter a value like 72057594037927935  to an INTEGER column, SQLite manager shows a wrong value (72057594037927940) instead. A different tool shows the correct value in the DB.

Table is created as: "CREATE TABLE "TEST" ("test" INTEGER PRIMARY KEY  NOT NULL  DEFAULT 0)"

Even if I enter 72057594037927935 and 72057594037927936 to the same table, SQLite Manger is showing the value 72057594037927940 twice.

Used versions are SQLite Manager 0.8.3, Firefox 31.6, Win7 64-bit.

**Comment by silverbacknet** [comment]
Yeah, that's pretty much the oldest bug of the project: https://web.archive.org/web/20130619103706/http://code.google.com/p/sqlite-manager/issues/detail?id=3
